src/js/lib/monster.apps.js

```diff
@@ -624,9 +610,10 @@ define(function() {
 							language = _.find([
 								monster.config.whitelabel.language,
 								monster.defaultLanguage
-							], _.partial(_.includes, loadedLanguages));
+							], _.partial(_.includes, loadedLanguages)),
```

Comment:
The problem with this condition is that the variable `loadedLanguages` is set after `app.data.i18n` is merged with the core app's i18n object above in [line 603](https://github.com/2600hz/monster-ui/pull/1119/files#diff-ba22e5d634130f5cee1042a194d3b9d7744331d3df283eae651061c58d326242R603)

I think it would be better to set `loadedLanguages` at [line 601](https://github.com/2600hz/monster-ui/pull/1119/files#diff-ba22e5d634130f5cee1042a194d3b9d7744331d3df283eae651061c58d326242R601) so it only includes the languages of the target app, so this way the parameter `languageSupported` won't be needed, and the code will work as originally intended.